delete_libvirt_vms.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: kvm
```

Comment:
Can we rename the playbook to "delete_libvirt.yml"? It's a detail, we are not deleting the vms only but also the volumes.